Reference docs: default timeout for scripts and software (#22908)
fleetdm/fleet: docs/REST API/rest-api.md

The new script will run after other activities finish. Failure of one activity won't cancel other activities.

> If the script run takes longer than 5 minutes (default), Fleet will stop the script. You can modify this default in your [agent configuration](https://fleetdm.com/docs/configuration/agent-configuration#script-execution-timeout).

Reviewer: Consider simplifying to:
```suggestion
By default, script runs time out after 5 minutes. You can modify this default in your [agent configuration](https://fleetdm.com/docs/configuration/agent-configuration#script-execution-timeout).
```

Install software (package or App Store app) on a macOS, iOS, iPadOS, Windows, or Linux (Ubuntu) host. Software title must have a `software_package` or `app_store_app` added to be installed.

> If the software install takes longer than 1 hour, Fleet will stop the install.

Reviewer 2: :+1: that's correct, only thing to keep in mind (worth mentioning?) is that this only applies to software packages (i.e. installs that use scripts), we don't have control on VPP app installation as this is an MDM-controlled command, we just send the command and Apple's MDM protocol takes care of the rest.

Author: Ah, thanks Martin. It makes sense to be explicit here.

I updated the language to "custom package" instead of "software"
